timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/server/endpoints/v2/Search.java

```diff
@@ -117,6 +126,27 @@ public Response get(@PathParam("id") UUIDParam id,
       .build();
   }
 
+  @GET
+  @Path("{id}/xls")
+  @Produces("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+  public Response get(@PathParam("id") UUIDParam id, @QueryParam("depth") @DefaultValue("1") int depth,
+                      @QueryParam("types") List<String> relationNames) {
+
+    Optional<SearchResult> searchResult = searchStore.getSearchResult(id.get());
+    SXSSFWorkbook workbook = new SXSSFWorkbook();
+    if (searchResult.isPresent()) {
+      final SearchResult result = searchResult.get();
+      workbook = excelExportService
+        .searchResultToExcel(result.getSearchResult(), result.getSearchDescription().getType(), depth, relationNames);
+    } else {
+      workbook.createSheet("result").createRow(0).createCell(0).setCellValue("Search with id " + id + " not found.");
```

Comment:
Return a 404 NOT_FOUND when the search result can not be found. This is how Timbuctoo v3 works.